testing/plugins_integration/requirements.txt

```diff
@@ -4,7 +4,9 @@ pytest-asyncio==0.16.0
 pytest-bdd==5.0.0
 pytest-cov==3.0.0
 pytest-django==4.4.0
-pytest-flakes==4.0.4
+# See https://github.com/asmeurer/pytest-flakes/pull/45
```

Comment:
Needed a new PR on `pytest-flakes`. 👍

Reply:
Given that @asmeurer plans a release according to https://github.com/asmeurer/pytest-flakes/pull/45#issuecomment-984171700, should we wait for that, so we can use `==4.0.5` or whatever here instead?

Reply:
Yeah that's my plan. 👍

Reply:
Done!